src/js/ripple/amount.js

```diff
@@ -1,12 +1,16 @@
+ 'use strict';
+
+ /*eslint-disable new-cap*/
```

Comment:
Yeah, I think it would be better to get rid of it, it doesn't seem like it should be a public thing. It might be nice to add a parameter to the amount constructor to make it explicit that we are setting it to NaN.